styles.css

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 16px;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 20px;
+}
+
+div.logo h3 {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+}
+
+div.hamburger {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+div.hamburger span {
+  height: 2px;
+  background: #6070ff;
+  width: 16px;
+  display: inline-block;
+  margin: 2px 0;
+}
+
+#headline {
+  padding: 114px 24px;
+  background: url("images/Header\ bg@2x.png") no-repeat;
+  background-size: cover;
+  background-position: center top;
```

Comment:
- [ ] The background image should be aligned properly. Aligning them properly according to the original design on Figma makes your app prettier. Kindly note the numbers 1, 2 and 3 in the image attached below and compare them to the background image you have. 

![screenshot](https://user-images.githubusercontent.com/77106962/204925518-1ddca0c6-be6e-4b00-81f5-948d281cfcdc.png)

#### Hint: Numbers 1 and 2 are missing in your app and number 3 is not properly aligned. 👍